Enhance and refactor project to a simpler and customizable direction (#50)
Ilshidur/action-slack: CUSTOM_PAYLOAD.md

> # Custom payload

Reviewer: ```suggestion
# 🚀 Slack for GitHub Actions - CUSTOM PAYLOAD
```

Custom payload allows interpolate hook data and event payload

> #### example

Reviewer: ```suggestion
## Example
```

```yml
  env:
    SLACK_CUSTOM_PAYLOAD: '{"text":"Custom payload from {{ GITHUB_REPOSITORY }} \\n *with* `new` _line_ escaped","username": "{{ GITHUB_ACTOR }}"}'
```

> ## IMPORTANT:

> >   - escaped characteres (like: line-breaks) needs to be escaped 2x: <br/> ex: `\n` becomes: `\\n`

Reviewer: ```suggestion
>   - Escaped characters (like: line-breaks) needs to be escaped 2x: <br/> ex: `\n` becomes: `\\n`
```

>   - Mind to send a quoted JSON string payload as the example above.

> >   - if using custom payload the other slack env vars data will be ignored.

Reviewer: ```suggestion
>   - Using a custom payload will cause the other slack env vars data to be ignored.
```
